pupil-spa/src/app/connectivity-error/connectivity-error.component.html

```diff
@@ -0,0 +1,57 @@
+<app-header class="full-width"></app-header>
+<main id="content" class="js-content" role="main">
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <header class="page-header">
+        <h1 tabindex="0">Loading error</h1>
+      </header>
+      <br />
+      <div class="notice">
+        <strong class="visually-hidden">Warning</strong>
+        <app-svg-warning></app-svg-warning>
+        <strong>
+          Please tell your teacher
+        </strong>
+      </div>
+      <br />
+      <details>
+        <summary><span class="summary">Instructions for teachers</span></summary>
+        <div class="panel no-border">
+          <p>{{serviceErrorMessage}}</p>
+          <p>Network administrators can apply restrictive internet settings to pupil devices so only selected trusted websites can be accessed.
+            These settings could prevent school devices from logging into the multiplication tables check pupil application.
+          </p>
+          <ol class="list list-number">
+            <li>
+              Advise your school’s internet network team to ensure the following are accessible or whitelisted:
+              <div>
+                *.windows.net - for data submission <br />
+                *.azurewebsites.net - for pupil authentication
+              </div>
+              <br />
+              <p>
+                Advise your schools administrators to enable the schools network and web browser configuration to allow this website to communicate securely over the TLS 1.2 protocol.
```

Comment:
Shouldn't this be the 'school‘s administrators'?

Reply:
@jon-shipley replaced.